draft-ietf-rats-corim.md

```diff
@@ -1066,6 +1066,182 @@ The following describes each member of the `concise-bom-tag` map.
+- The TcbInfo `vendor` field SHALL be copied to the field named `environment-map / class / vendor`
+- The TcbInfo `model` field SHALL be copied to the field named `environment-map / class / model`
+- The TcbInfo `layer` field SHALL be copied to the field named `environment-map / class / layer`
+- The TcbInfo `index` field SHALL be copied to the field named `environment-map / class / index`
+
+- The TcbInfo `version` field SHALL be translated to the field named `measurement-map / mval / version / version`
+- The TcbInfo `svn` field SHALL be copied to the field named `measurement-map / mval / svn`
+- The TcbInfo `fwids` field SHALL be translated to the field named `measurement-map / mval / digests`
+  - Each digest within fwids is translated to a CoMID digest object, with an appropriate algorithm identifier
+- The TcbInfo `flags` field SHALL be translated to the field named `measurement-map / mval / flags`
+  - Each flag is translated independently 
+- The TcbInfo `vendorInfo` SHALL shall be copied to the field named `measurement-map / mval / raw-value`
+
+If there are multiple evidence triples with the same environment map then ...
+
+## Evidence appraisal
+
+In the Evidence Appraisal phase, a CoRIM Appraisal Context and an Evidence Appraisal Policy are used to convert the received Evidence from its raw form into a more usable form. This phase may be repeated multiple times .
+The outcome of the appraisal process is summarised in an Attestation Result.
+The Relying Party application uses the content of the Attestation Result to make its own policy decisions.
+
+We make no assumptions on the specific shape of the Attestation Result, except for its optional ability to include Evidence from the attestor and Endorsed Values that the Verifier has been able to infer from Evidence and the Appraisal Context.
+
+> A CoRIM profile MUST describe:
+>
+> * How Reference Values are conveyed (e.g., using Reference Value triples, or CoSWID tags)
+> * How Reference Values are associated with the Target Environment
+> * How the Target Environment is identified in Evidence
+> * How measurements from Evidence are matched against Reference Values
+
+> A CoRIM profile MUST describe:
+>
+> * How Endorsed Values are conveyed (e.g., using Endorsed Value triples, or Device Identity triples)
+> * How Endorsed Values are associated with the Target and Attesting Environments
+> * How the Target and Attesting Environments are identified in Evidence
+
+The sections below describe the algorithm used to compare CoMID tags.
+
+### Comparing and processing CoMID tags
+
+There can be multiple passes of the CoMID evidence appraisal process.
+
+In each pass, each potential CoMID tag in the Appraisal Context is compared against the accepted claims set (which has been initialised with Evidence from the attestor(s) ). If the reference values in the CoMID tag match the Accepted Claims Set then Endorsements and other values from the tag are added to the Accepted Claims Set.
```

Comment:
Note: this process assumes the Evidence, Reference Values and Endorsed Values have a common environment.